src/ui/components/statusrow/StatusRow.js

```diff
@@ -1,54 +1,78 @@
 const STATUS_KEYS = {
     OUT_OF_SERVICE: "OUT_OF_SERVICE",
     EIS: "EIS",
     RADIOACTIVE: 'RADIOACTIVE',
+    SAFETY_CONFORMITY: 'SAFETY_CONFORMITY',
 }
 
 const iconStyle = {
     height: "25px",
     width: "25px"
 }
 
+const safetyConformity = ({
+    'CONFORME': { icon: <CheckCircleIcon style={{color: 'green', ...iconStyle}} />, description: 'Conform', tooltip: 'Conform' },
+    'CONFORME-RESERVES': { icon: <ReportProblemIcon style={{color: 'orange', ...iconStyle}}/>, description: 'Observations', tooltip: 'Observations' },
+    'OBSERVATIONS': { icon: <ReportProblemIcon style={{color: 'orange', ...iconStyle}}/>, description: 'Observations', tooltip: 'Observations' },
+    'NON CONFORME': { icon: <ReportIcon style={{color: 'red', ...iconStyle}}/>, description: 'NON conform', tooltip: 'NON conform' },
+    'NON INSPECTE': { icon: <ErrorOutlinedIcon style={{color: 'orange', ...iconStyle}}/>, description: 'Not inspected', tooltip: 'Not inspected' },
+});
+
+const getSafetyConformity = (entity) => {
+    return safetyConformity[entity.userDefinedFields.udfchar30];
+};
+
 const STATUSES = [
     {
         key: STATUS_KEYS.OUT_OF_SERVICE,
         shouldRender: (entity, entityType) => entity.outOfService === "true" || entity.outOfService === true,
-        icon: <BlockIcon style={{color: 'red', ...iconStyle}}/>,
-        description: "Out of Service",
-        tooltip: "Out of Service",
+        getIcon: () => <BlockIcon style={{color: 'red', ...iconStyle}}/>,
+        getDescription: () => "Out of Service",
+        getTooltip: () => "Out of Service",
     },
     {
         key: STATUS_KEYS.EIS,
         shouldRender: (entity, entityType) => isCernMode && entityType === "equipment" && entity.userDefinedFields.udfchkbox01 === "true",
-        icon: <EISIcon style={{color: 'blue', ...iconStyle}}/>,
-        description: "EIS",
-        tooltip: "Élément Important pour la Sécurité",
+        getIcon: () => <EISIcon style={{color: 'blue', ...iconStyle}}/>,
+        getDescription: () => "EIS",
+        getTooltip: () => "Élément Important pour la Sécurité",
     },
     {
         key: STATUS_KEYS.RADIOACTIVE,
         shouldRender: (entity, entityType) => isCernMode && entityType === "equipment" && entity.userDefinedFields.udfchar04 === "Radioactive",
-        icon: <RadioactiveWarningIcon style={iconStyle}/>,
-        description: "Radioactive",
-        tooltip: "Radioactive",
-    }
-
+        getIcon: () => <RadioactiveWarningIcon style={iconStyle}/>,
+        getDescription: () => "Radioactive",
+        getTooltip: () => "Radioactive",
+    },
+    {
+        key: STATUS_KEYS.SAFETY_CONFORMITY,
+        shouldRender: (entity, entityType) => isCernMode && entityType === "equipment" && !isEmpty(entity.userDefinedFields.udfchar30),
+        getIcon: (entity) => getSafetyConformity(entity).icon,
+        getDescription: (entity) => getSafetyConformity(entity).description,
+        getTooltip: (entity) => getSafetyConformity(entity).tooltip,
```

Comment:
If this check is not performed, `getSafetyConformity(entity)` will return undefined here and crash the application when we try to access one of its properties.